Cargo.toml

```diff
@@ -63,6 +63,7 @@ unicode-xid-ident = ["unicode-xid"] # allow Unicode Standard Annex #31 for ident
 metadata = ["serde", "serde_json", "rhai_codegen/metadata", "smartstring/serde"] # enable exporting functions metadata
 internals = []                  # expose internal data structures
 debugging = ["internals"]       # enable debugging
+stable_hash = ["std"]
```

Comment:
Is there a way to avoid introducing _yet_ another feature flag?  Perhaps gate it under `nightly`?

Reply:
I don't think so - there isnt a `rustc_version` attribute unless you are willing to do build.rs trickery or pulling another crate. Keep in mind that this is incompatible with your no-std-compat crate as that one does not ship nightly features, so keeping it feature gated even in nightly is probably a good idea.